### DISCUSSION TITLE
joint.shapes.uml is undefined

### DISCUSSION THREAD

**Discussion by makleo2002** [clientIO/joint#2640] [Q&A]
### Introduction

I installed joint-plus, which still has Joint.core. Why is there an error when I try to create a shape from a uml package:
joint.shapes.uml is undefined

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
The `uml` package does not exist. We do not distribute `uml` shapes as part of JointJS.

There are several demos that implement UML shapes. These shapes are defined in the example itself.
https://www.jointjs.com/demos?*=uml
